push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/NotificationRouter.java

```diff
@@ -55,7 +56,7 @@
     private final AeroGearLogger logger = AeroGearLogger.getInstance(NotificationRouter.class);
 
     @Inject
-    private GenericVariantService genericVariantService;
+    private Instance<GenericVariantService> genericVariantService;
```

Comment:
it was causing a test error - the mock genericVariantService was stored as `@RequestScoped` but the bean was seeing the cached version - I'm not sure why, but using `Instance#get()` refreshed the instance